client/js/components/map/map/DpOlMap.vue

```diff
@@ -356,7 +356,7 @@ export default {
       })
         .then(checkResponse)
         .then(response => response.data.attributes)
-        .catch(error => checkResponse(error.response))
+        .catch(error => checkResponse(Translator.trans(error.response)))
```

Comment:
`checkResponse` expects one or two objects as arguments, not a string. Did you test this?

Reply:
Here nothing has to change. You pass the response object as before. Only in cases, where two params are in use. Then You have to think about how to change the later one. But I haven't seen any here in the PR.

Reply:
https://github.com/demos-europe/demosplan-core/pull/3467/commits/8dc449f99902825c72d351d2e70f509398f6a709

Reply:
I just want to point out that whether the messages are translated already in the response or not, what we pass here to `checkResponse()` is an object, on which we cannot use `Translator.trans()` in general. The translations happen inside `checkResponse`/`handleResponseMessages`.